letta/config.py

```diff
@@ -45,7 +45,7 @@ def set_field(config, section, field, value):
 @dataclass
 class LettaConfig:
     config_path: str = os.getenv("MEMGPT_CONFIG_PATH") or os.path.join(LETTA_DIR, "config")
-    anon_clientid: str = DEFAULT_USER_ID
+    anon_clientid: str = UserManager.DEFAULT_USER_ID
```

Comment:
I think we should just delete `anon_clientid`, and keep user/org ID + name as constants

Reply:
i'm going to do this in a follow up PR